terraform/fec_rds.tf

```diff
@@ -93,13 +93,21 @@ resource "aws_db_instance" "rds_production" {
   publicly_accessible = true
   storage_encrypted = true
   multi_az = true
+  auto_minor_version_upgrade = true
+  storage_type = "io1"
+  identifier = "fec-govcloud-prod"
```

Comment:
Ah right, good catch!  I was changing them here because we were seeing strings with just random alphanumeric characters in them for the names when we looked in the console, which were not all that clear in terms of what was what.

@LindsayYoung, do you think we should go ahead and change them?  If so, now is definitely the time to do it as opposed later on, I would think (except for staging at the moment due to the aforementioned ATO reason).